hw1/02-palindrome.js

```diff
@@ -1,3 +1,23 @@
 const elem = document.querySelector('input');
-
+const resultText = document.getElementById('resultText');
 elem.addEventListener('input', handleInput);
+
+function handleInput(){
+    let input = parseInt(elem.value);
+    if(input >= 0){
+        let reverseInput = parseInt(String(elem.value).split('').reverse().join(''));
+        if( input == reverseInput){
```

Comment:
The Airbnb JavaScript Style Guide suggests using "===" instead of "==" because "===" checks the value and type of what you're comparing, making it less likely to cause mistakes.

Reply:
Done!